### ISSUE TITLE
Some CODEOWNERS lack write access, breaking CODEOWNERS functionality.

### ISSUE THREAD

**Issue by SamuelTrahanNOAA** [ufs-community/ccpp-physics#7]
For the CODEOWNERS functionality to work, all accounts in the CODEOWNERS file must have write access to this repository. That means you need to do one of these:

1. Remove people from CODEOWNERS.
2. Delete CODEOWNERS.
3. Grant accounts in CODEOWNERS write access.
4. Ignore the problem, and have CODEOWNERS continue to be non-functional, but still match the NCAR ccpp-physics CODEOWNERS file.

If you want the repository to be identical to NCAR ccpp-physics, but don't want CODEOWNERS to work, then option 4 is the best way. In that case, you should give this bug the "will not fix" label.

**Comment by dustinswales** [comment]
@SamuelTrahanNOAA Thanks for identifying this.
@grantfirl @ChunxiZhang-NOAA We should discuss adding the POCs from CODEOWNERS in ufs-community?

**Comment by SamuelTrahanNOAA** [comment]
They'll need write access, which gives them broad powers. I haven't seen any of them break anything on NCAR's repository, so I'm not personally worried. It's not my repository though; you're taking the risks.

**Comment by ChunxiZhang-NOAA** [comment]
@grantfirl @dustinswales Yes. I think the POCs in the ufs-community should be the same as those in other model-community. We can discuss it in this Wednesday's meeting.

**Comment by ligiabernardet** [comment]
Also, we (perhaps that falls on me) need to inform the POCs of this new responsibility and figure out how to avoid them reviewing the same code twice (PR to the fork, then PR to the authoritative).
